### PR TITLE
Object detection op with fasterrcnn

### PR DESCRIPTION
**Pull request by derekdqc**
mainly add files:
towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py
towhee/tests/operators/test_pytorch_object_detection_operator.py

### CODE REVIEW

File: towhee/tests/mock_operators/__init__.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 SUB_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "sub_operator")
 PYTORCH_CNN_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_cnn_operator")
 PYTORCH_IMAGE_CLASSIFICATION_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_image_classification_operator")
+PYTORCH_OBJECT_DETECTION_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_object_detection_operator")
```
Comment (fzliu) [inline towhee/tests/mock_operators/__init__.py:29]: Let's put new operators in an `operators` folder under the Towhee root. We will move operators used for tests to this root folder later on.

File: towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import numpy
+import torch
+
+from typing import NamedTuple, List
+from pathlib import Path
+
+import torchvision
+
+from towhee.operator import Operator
+
```
Comment (fzliu) [inline towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py:23]: For import order, let's do:
```
import <system_libraries_first>

import <pip_installed_libraries_second>

import <towhee_libraries_last>
```

File: towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    'train', 'truck', 'boat', 'traffic light', 'fire hydrant', 'N/A', 'stop sign',
+    'parking meter', 'bench', 'bird', 'cat', 'dog', 'horse', 'sheep', 'cow',
+    'elephant', 'bear', 'zebra', 'giraffe', 'N/A', 'backpack', 'umbrella', 'N/A', 'N/A',
+    'handbag', 'tie', 'suitcase', 'frisbee', 'skis', 'snowboard', 'sports ball',
+    'kite', 'baseball bat', 'baseball glove', 'skateboard', 'surfboard', 'tennis racket',
+    'bottle', 'N/A', 'wine glass', 'cup', 'fork', 'knife', 'spoon', 'bowl',
+    'banana', 'apple', 'sandwich', 'orange', 'broccoli', 'carrot', 'hot dog', 'pizza',
+    'donut', 'cake', 'chair', 'couch', 'potted plant', 'bed', 'N/A', 'dining table',
+    'N/A', 'N/A', 'toilet', 'N/A', 'tv', 'laptop', 'mouse', 'remote', 'keyboard', 'cell phone',
+    'microwave', 'oven', 'toaster', 'sink', 'refrigerator', 'N/A', 'book',
```
Comment (fzliu) [inline towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py:37]: Will these N/A values interfere with output results?

File: towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+]
+lb_names = COCO_INSTANCE_CATEGORY_NAMES
+
+
+class PyTorchObjectDetectionOperator(Operator):
+    """
+    PyTorch model operator base
+    """
+
+    def __init__(self, model_name) -> None:
```
Comment (fzliu) [inline towhee/tests/mock_operators/pytorch_object_detection_operator/pytorch_object_detection_operator.py:48]: Add some docstrings here for easy of understanding.

**Review by junjiejiangjjj — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**